python/pdo/service_client/__init__.py

```diff
@@ -17,5 +17,4 @@
     'generic',
     'provisioning',
     'storage',
-    'servicedatabase'
 ]
```

Comment:
why did you remove this?

Reply:
this should not have been ever there.

(we originally had the database part of pdo/service_client, so this was added, but then created a new folder service_data (which has an init for the database). so this ought have been removed.